mobile-v3/ios/Flutter/Generated.xcconfig

```diff
@@ -1,14 +1,14 @@
 // This is a generated file; do not edit or check into version control.
 FLUTTER_ROOT=/Users/jordanojangole/dev/flutter
-FLUTTER_APPLICATION_PATH=/Users/jordanojangole/dev/projects/airqo
+FLUTTER_APPLICATION_PATH=/Users/jordanojangole/dev/projects/AirQo-frontend/mobile-v3
 COCOAPODS_PARALLEL_CODE_SIGN=true
 FLUTTER_TARGET=lib/main.dart
 FLUTTER_BUILD_DIR=build
 FLUTTER_BUILD_NAME=3.0.2
-FLUTTER_BUILD_NUMBER=0
+FLUTTER_BUILD_NUMBER=2
 EXCLUDED_ARCHS[sdk=iphonesimulator*]=i386
 EXCLUDED_ARCHS[sdk=iphoneos*]=armv7
 DART_OBFUSCATION=false
-TRACK_WIDGET_CREATION=true
-TREE_SHAKE_ICONS=false
-PACKAGE_CONFIG=.dart_tool/package_config.json
+TRACK_WIDGET_CREATION=false
+TREE_SHAKE_ICONS=true
+PACKAGE_CONFIG=/Users/jordanojangole/dev/projects/AirQo-frontend/mobile-v3/.dart_tool/package_config.json
```

Comment:
these package configs seems to be very personal (auto generated) in nature? Do we want to deleted this file and put it inside .gitignore? @jordanopos